### DISCUSSION TITLE
Internet Access

### DISCUSSION THREAD

**Discussion by Lio-gif** [Taewan-P/gpt_mobile#146] [Ideas]
Add a Feature to let the Models use the internet.
![IMG_20241229_000429](https://github.com/user-attachments/assets/6d0a4356-a3c5-4153-820d-a8b223f2855b)
ChatGpt is trying but its Not working.

**Comment by Taewan-P** [comment]
AFAIK, internet access is not available in APIs. I think you need to implement a web search api with function calling. See here: https://community.openai.com/t/gpt4-o-can-connect-to-the-internet-via-the-api/769981/2
